assignment4.js

```diff
@@ -0,0 +1,16 @@
+let files = ["./f1","./f2","./f3", "./f4", "./f5"], i = 0;
+fs = require('fs');
```

Comment:
`fs` should be defined. using global namespace is not supported by all NodeJS version.